### PR TITLE
Create backup-private-key guide

### PR DESCRIPTION
**Pull request by biancabuzea200**
this still needs to be checked on the recommendations for keeping the JSON file secure

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-5564 enhancing documentation on Metamask-UP conflict](https://app.clickup.com/t/2645698/DEV-5564)

File: docs/guides/browser-extension/sign-in-with-ethereum.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_label: 'Sign-In with Ethereum'
-sidebar_position: 4
+sidebar_position:
```
Comment (Hugoo) [inline docs/guides/browser-extension/sign-in-with-ethereum.md:3]: why?

This breaks the CI:

image

**Review by Hugoo — COMMENTED** [review]
ah my bad this is draft ;)

File: docs/guides/browser-extension/sign-in-with-ethereum.md (hunk 1 above)
Comment (biancabuzea200) [reply docs/guides/browser-extension/sign-in-with-ethereum.md]: Should be fine now, just forgot a position number for this page

**Review by Hugoo — APPROVED** [review]

**Pull request merged**